src/Database/Db.php

```diff
@@ -190,7 +190,7 @@ public static function __callStatic(string $method, $arguments)
  * @param array $values An array of values which should be inserted
  * @return int ID of the inserted row
  */
-Db::$queries['insert'] = function (string $table, array $values): int {
+Db::$queries['insert'] = function (string $table, array $values) {
```

Comment:
This line uncovered.

Reply:
I guess we need `@codeCoverageIgnoreStart` and `@codeCoverageIgnoreEnd` for inline codes 🤔

Reply:
I think Ahmet is right, the `@codeCoverageIgnore` seems to not be applied at all to these inline closures, @lukasbestle .